pkg/kubelet/prober/prober_manager.go

```diff
@@ -48,6 +48,22 @@ var ProberResults = metrics.NewCounterVec(
 		"pod_uid"},
 )
 
+// ProberDuration stores the duration of a successful probe lifecycle by result as prometheus metrics.
+var ProberDuration = metrics.NewHistogramVec(
+	&metrics.HistogramOpts{
+		Subsystem:      "prober",
+		Name:           "probe_duration",
```

Comment:
Histogram metrics should be suffixed by the unit of the metric. In this case, it should be:
```suggestion
		Name:           "probe_duration_seconds",
```